linera-base/build.rs

```diff
@@ -8,6 +8,7 @@ fn main() {
         with_metrics: { all(not(target_arch = "wasm32"), feature = "metrics") },
         with_reqwest: { feature = "reqwest" },
         with_testing: { any(test, feature = "test") },
+        with_revm: { any(test, feature = "revm") },
```

Comment:
I find it a bit surprising that `with_revm` means "with `revm` feature or `test` target". Maybe `with_revm_or_test` then? Or just make the tests depend on the `revm` flag?